wmf2svg/src/net/arnx/wmf2svg/gdi/svg/SvgGdi.java

```diff
@@ -1187,6 +1307,12 @@ public void setViewportOrgEx(int x, int y, Point old) {
 	}
 
 	public void setWindowExtEx(int width, int height, Size old) {
+		if (width > maxWidth) {
+			maxWidth = width;
+		}
+		if (height > maxHeight) {
+			maxHeight = height;
+		}
 		dc.setWindowExtEx(width, height, old);
```

Comment:
maxWidth, maxHeight value need to be taken from dc because it restore or save

Reply:
If you could please explain in little detail, that shall be very helpful.